### PR TITLE
logging: enable log entry with line number in debug mode

### PR DESCRIPTION
**Pull request by schen59**

### CODE REVIEW

File: etcdmain/etcd.go [hunk 1]
```diff
@@ -520,6 +520,8 @@ func identifyDataDirOrDie(dir string) dirType {
 func setupLogging(cfg *config) {
 	capnslog.SetGlobalLogLevel(capnslog.INFO)
 	if cfg.debug {
+		formatter := capnslog.NewPrettyFormatter(os.Stdout, true)
```
Comment (jonboulle) [inline etcdmain/etcd.go]: Why stdout?

Comment (schen59) [inline etcdmain/etcd.go]: Stderr is used by capnslog's default formatter. Thought Stdout is more appropriate for log output. Not quite sure.

Comment (jonboulle) [inline etcdmain/etcd.go]: Nope, all logging should go to stderr.

Comment (schen59) [inline etcdmain/etcd.go]: changed to stderr.

**Comment by xiang90** [comment]
We need to be init aware.

Please check https://github.com/coreos/pkg/blob/master/capnslog/init.go#L40-L49.

@gyuho Do you feel we need to log the line? I do not have strong feeling on this one.

**Comment by gyuho** [comment]
@xiang90 I personally don't need line number even in debugging mode, but I think some people should find it useful.

**Comment by heyitsanthony** [comment]
is this still worth having? /cc @gyuho @xiang90

**Comment by gyuho** [comment]
@heyitsanthony No. Let's close this.

**Comment by xiang90** [comment]
i do not feel so.

**Pull request closed without merging**